pallets/msa/src/tests.rs

```diff
@@ -2693,57 +2693,6 @@ fn revoke_permissions_for_schemas_errors_when_schema_does_not_exist_in_list_of_s
 	});
 }
 
-#[test]
```

Comment:
Are these not needed anymore?

Reply:
Correct.  It has been deprecated.  Although I should have indicated that.